ocean-client/src/utils/store_config.ts

```diff
@@ -93,9 +104,13 @@ class ConfigFile {
   seedfile: string = 'V:/store/vault-maxi-seed.txt'
   minCollateralRatio: number = 200
   maxCollateralRatio: number = 250
-  LMToken: string = 'GLD'
-  LMPair: string | undefined
+  LMToken: string | undefined
+  LMPair: string = 'GLD-DUSD'
```

Comment:
why the change? to be backwards compatible, logic is "if LMPair is not there -> take LMToken". if you give LMPair a default value, its always there and therefore the logic is broken.

Reply:
This is only for initial config file creation, it creates the file now with the predefined pair instead of lmtoken. I wanted to have it like in aws, there's also only a pair setting. Both works here lmtoken or pair in the settings file.

Reply:
how should it look like? like red lines 96 97 or the green ones 107 108 ?

Reply:
IMHO it should be the red lines. for backward compatibility